method.h

```diff
@@ -179,10 +179,9 @@ typedef struct rb_method_optimized {
 struct rb_method_definition_struct {
     BITFIELD(rb_method_type_t, type, VM_METHOD_TYPE_MINIMUM_BITS);
     unsigned int iseq_overload: 1;
-    int alias_count : 27;
-    int complemented_count : 28;
     unsigned int no_redef_warning: 1;
-
+    unsigned int aliased : 1;
+    int reference_count : 28;
```

Comment:
Nit: can you add a newline here?

```suggestion
    int reference_count : 28;

```